recipe/meta.yaml

```diff
@@ -24,7 +24,8 @@ requirements:
   run:
     - python
     - spyder >=3.2
-    - pweave
+    - pweave 0.25  # [py27]
```

Comment:
@ccordoba12, is not it better to use `pweave <=0.25` ? it has to be that specific version?

Reply:
There are no previous versions of Pweave here on conda-forge, and that's the last version to work with Python 2.

Reply:
Ok!